firefox_ios/views/counters.view.lkml

```diff
@@ -137,6 +136,16 @@ view: counters {
     suggest_dimension: metrics_sample.app_display_version
   }
 
+  dimension: app_display_major_version {
+    type: number
+    sql: mozfun.norm.truncate_version(${app_display_version}, "major") ;;
+  }
+
```

Comment:
Looks like `mozfun.norm.truncate(x, "minor")` will pull out the major and minor versions, so we probably only need one field with the minor version below.
```suggestion
```

Reply:
I don't think this is necessarily true, I think having just the major version is useful too.

Reply:
I think I'd rather people get comfortable using more generic fields, but if you think it's good for this instance then we should keep it.

Reply:
I don't think there's a way to round a dimension, if you only wanted to see major version over time. The major-minor versions can be too noisy.